client_delivery/shared/src/commonMain/kotlin/presentation/login/composable/LogoHeaderCard.kt

```diff
@@ -0,0 +1,57 @@
+package presentation.login.composable
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.unit.dp
+import com.beepbeep.designSystem.ui.theme.Theme
+import org.jetbrains.compose.resources.ExperimentalResourceApi
+import org.jetbrains.compose.resources.painterResource
+import resources.Resources
+
+@OptIn(ExperimentalResourceApi::class)
+@Composable
+fun LogoHeaderCard(
```

Comment:
I don't see this is considered as a component and should be separated, don't you think so? Can you use it outside the  BeeBeebApp? no! So it is not a component